staging/src/k8s.io/apiserver/pkg/storage/etcd3/watcher.go

```diff
@@ -209,6 +224,11 @@ func (wc *watchChan) startWatching(watchClosedCh chan struct{}) {
 			wc.sendError(err)
 			return
 		}
+
+		if wres.IsProgressNotify() && wc.watcher.watchBookmarkEnabled {
+			wc.sendEvent(progressNotifyEvent(wres.Header.GetRevision()))
```

Comment:
continue here? no need to go through wres.Events for progressNotification

Reply:
It is empty there, but add a `continue` for clarity.